src/util/nft.js

```diff
@@ -74,12 +84,20 @@ export async function sendGrant({ senderAddress, amountInLIKE, signer }) {
     },
   ];
   const expirationInMs = Date.now() + 1000 * 90;
-  const { transactionHash } = await client.createSendGrant(
+  const signData = await client.createSendGrant(
     senderAddress,
     LIKECOIN_NFT_API_WALLET,
     spendLimit,
-    expirationInMs
+    expirationInMs,
+    { broadcast: false }
   );
+  return signData;
+}
+
+export async function broadcastTx(signData, signer) {
+  const txBytes = TxRaw.encode(signData).finish();
+  const senderClient = await getSenderClientBySigner(signer);
```

Comment:
I think this can be replaced by
https://github.com/likecoin/liker-land/blob/f9fad1b2c0d400ee7b1638c28c1b5142fec1a461/src/util/nft.js#L23-L27

```suggestion
  const client = await createNFTSigningClient(signer);
  const senderClient = client.getSigningStargateClient();
```